### ISSUE TITLE
Gather help desk questions for Contact Center agents

### ISSUE THREAD

**Issue by joanneesteban** [department-of-veterans-affairs/va.gov-team#6957]
## User Story or Problem Statement
As a VFS apps team, I need to better understand veteran issues with my website so I can research & resolve the issues more efficiently.

## Goal
Gather VFS team technical questions and disseminate to VEO Call Center.

## Acceptance Criteria
- [ ] Gather questions from VFS teams
    - [x] 8 VSA teams
    - [x] VAOS
    - [ ] Education
    - [x] SSO/Login
- [x] Consolidate into one doc
- [ ] Send to Todd
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by Kimberley2019** [comment]
Slacked the following message on 3/18/2020:

_Good_ Afternoon Team,

I am one of the Contact Center Leads that works with Contact/Calls Centers and the issues that are being reported by users of VA.gov.
In an effort to maximize the information that we receive from them, I am politely requesting that your team take a few moments to determine if there are any specific questions that should be asked by the agents while they are in communication with the caller concerning their issue.

For example:  If a user is calling in to report a problem uploading a document; what are some of the details that your team would need in order to more efficiently research and resolve the issue that they are experiencing?

Please keep in mind two important things:
Some users (and agents) are not as technically advanced as you or some of your counterparts, so try phrase the questions in layman terms.  This will help ensure that all individuals understand what they are asking and what is being asked of them.
Try to keep the questions to a minimum as much as possible.  While we want to gather as much information as we can for your team, we want to remain vigilant of SLA's and metrics that the Contact/Call Center has agreed to.  With this in mind, try to also incorporate questions that will determine if this is an actual issue that they may be experiencing or if it is actually a user error.

Hoping to have this in the next week or so so that call center agents can have this information to support the VFS applications sooner.  This information can be provided via the #vsp-contact-center-support channel.

I look forward to hearing from you in the near future._

Message was sent to:
- Authenticated Experience
- Benefits and Memorials
- Benefits and Memorials 2
- eBenefits
- Facilities
- Public Websites
- VA Medical Center (VAMC)
- Caregiver

**Comment by Kimberley2019** [comment]
Jason Wolf from eBenefits relayed:
One of the features we recently launched on VA.gov: "View your VA disability ratings"
- Are you able to login?  
- What method do you use?
- If you are getting an error message, when and where do you see it?
- When was the last time you saw a rating if none exists?
- Does the rest of the page load or does it say something over all unexpected?

This application currently uses EVSS and we would probably first check to see if that has failed in the event that an error message appeared unexpectedly.  I am looking into an easy way to do that now.
We also have a product guide and a video that dives into more of the functionality currently in place.
https://dsva.slack.com/files/ULW9ZB61X/F0109N6AQKA/rated-disabilities-call-center-walkthrough.mp4

**Comment by Kimberley2019** [comment]
Luke Majewski from benefits-memorials relayed:
We will have some discussions and respond back - thank you for taking the time to reach out.

**Comment by Kimberley2019** [comment]
Kelson Adams from public websites relayed:
For content-related tickets, we usually need:
1) the URL
2) the current content that's incorrect
3) how that content should be changed
4) any additional info you would like to provide that you think could be useful to accomplishing the ticket

**Comment by Kimberley2019** [comment]
Michelle Middaugh from Facilities relayed:
Hi @Kimberley.Daniels thanks for asking! I know we have a few things in mind already and I will work on putting this list together for you.
I am also in the process of working on a support doc for Facility Locator, fyi

**Comment by Kimberley2019** [comment]
Samara Strauss from authenticated experience relayed:
Thanks for this! Then I would say the main things we need to know regardless of product are:

- What the user is trying to accomplish
- What the problem is
- Step-by-step what the user did before their problem/question occurred
- Any errors or messaging they see

**Comment by Kimberley2019** [comment]
Alexis James from SSO relayed:
I'm on the SSO team and also responding to your request for questions last week. Please see the attached product guide. The main question to ask would be: "Please describe the error screen you are encountering". The product guide includes the possible error states which you can than communicate back to the team troubleshooting the issue. Please let me know if you have any questions. Thanks (edited) 

https://dsva.slack.com/files/UM8HUA64E/F0109SW5XK3/ssoe-product-guide.pdf

**Comment by Kimberley2019** [comment]
Michelle Middaugh from Facilities Locator relayed:

I'm responding to your message for Contact Center questions regarding the Facility Locator.

- Did you find what you were looking for? If not, what were you looking for and where did you expect to find it? (Or how did you try to find it?)

- If the search was successful, did the page include the information you needed? (If not, what was missing or inaccurate?)


- If the user reports inaccurate information (the hours of operation are not what they know them to be, for example), we have the following feedback options:

- For VA facilities (VHA, VBA, NCA and Vet Centers): please email the name of the Facility (and other available info) and the missing or inaccurate detail to Dave Conlon (david.conlon@va.gov) and myself (michelle.middaugh@adhocteam.us). 

- For community care providers and pharmacies, please email the name of the Facility (and other available info) and the missing or inaccurate detail to VHAOCCPPMSPROJBUSPOCS@VA.GOV, copying JR Ruble (Ronald.Ruble@va.gov), Dave Conlon (david.conlon@va.gov) and myself. (michelle.middaugh@adhocteam.us). 

- Also, the updated Product Guide can be found here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/contact-center.md

**Comment by Kimberley2019** [comment]
Luke Majewski from benefits-memorial relayed today:

- Get as much information about the error.  If possible, screenshot.  If possible, screenshot with computer time/clock or information about exactly when they submitted the claim.

- Name/SSN of veteran sent to Andrea (not written in ticket, obviously) so we can get the UUID.

- What type of document(s) were uploaded.   File types/extensions.  Did they try uploading one at a time?  
    Are any of the files (Word, PDF) locked with a password?

- Ask if the veteran will be willing to work with our software development team to help resolve the error 
    (only if the veteran actually got an error message)

**Comment by Kimberley2019** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/teams/insights-analytics/contact-center/technical-questions-for-agents.md
